content/comm/conferences/index.en.html

```diff
@@ -0,0 +1,323 @@
+<!DOCTYPE html>
```

Comment:
Could the html file be removed please? I think it's a by-product of a preview.

Reply:
Done!